frontend/src/views/WheresMyGene/components/HeatMap/components/XAxisChart/style.ts

```diff
@@ -17,6 +17,7 @@ export const XAxisMask = styled.div`
   width: ${Y_AXIS_CHART_WIDTH_PX}px;
   position: sticky;
   left: 0;
+  opacity: 0;
```

Comment:
Ah sorry the opacity is needed here for the sticky header behavior when the user adds enough genes and scrolling starts to happen!

To debug, we can make overflow happen sooner with less added genes by increasing the screen size with `cmd` + `+`

![mask opacity](https://user-images.githubusercontent.com/6309723/179625864-5a0181f0-6800-4dbe-998e-e09211025dbc.gif)

Reply:
Currently the `Cell Count` header is not sticky while the values are. I'm assuming the product requirement is wanting both to be sticky when scrolling?

If so, I'm thinking maybe we can create a new DOM element to display the Cell Count column without relying on echarts, and make sure the new element still covers the gene labels when scrolling happens?